climada/entity/exposures/litpop/nightlight.py

```diff
@@ -161,7 +164,7 @@ def check_nl_local_file_exists(required_files=np.ones(len(BM_FILENAMES),),
     for num_check, name_check in enumerate(BM_FILENAMES):
```

Comment:
Comment here because these are the lines that changed and github does not allow comments on other lines. The dcostring is incomplete.